main/lib/src/RunControl.cc

```diff
@@ -80,8 +82,12 @@ namespace eudaq {
     SendCommand("CONFIG", to_string(config));
     if (config.SetSection("RunControl")) {
       m_runsizelimit = config.Get("RunSizeLimit", 0LL);
+      m_runeventlimit = config.Get("RunEventLimit", 0LL);
+      m_nextconfigonfilelimit = config.Get("NextConfigFileOnFileLimit", false);
```

Comment:
FYI: Richard recently managed to use this feature (automatic stop configure start) by using a script something like...
'''
# changes DUT Mask from current value to 2
(Get-Content .\ni_autotrig.conf) -replace 'DutMask.*', 'DutMask = 2' | Set-Content .\ni_autotrig.conf -Encoding Ascii

# to change a parameter for all files in the folder:
ls *.conf |foreach{ (Get-Content $_) -replace 'DutMask.*', 'DutMask = 2' | Set-Content $_.FullName -Encoding Ascii }
'''
please refer to Richard for more details